client/src/components/logInForm.js

```diff
@@ -0,0 +1,98 @@
+import React, { useContext, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Form, Input, Button, Alert } from 'antd';
+import axios from 'axios';
+import IsAuthContext from './context/isAuthContext';
+
+const layout = {
+  labelCol: {
+    span: 8,
+  },
+  wrapperCol: {
+    span: 25,
+  },
+};
+
+const tailLayout = {
+  wrapperCol: {
+    offset: 8,
+    span: 16,
+  },
+};
+
+const LoginForm = () => {
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState('');
```

Comment:
make the default value as true then you don't need to setIsLoading(true) at line 31

Reply:
done